diff --git "a/Source/DataStructureAndEncodingDefinition/gdcmVM.h" "b/Source/DataStructureAndEncodingDefinition/gdcmVM.h"
index 700129746..dfa3b964e 100644
--- "a/Source/DataStructureAndEncodingDefinition/gdcmVM.h"
+++ "b/Source/DataStructureAndEncodingDefinition/gdcmVM.h"
@@ -98,6 +98,7 @@ public:
     VM1_n  = VM1 | VM2 | VM3 | VM4 | VM5 | VM6 | VM8 | VM9 | VM16 | VM24 | VM32 | VM99 | VM256,
     VM2_2n =       VM2       | VM4       | VM6 | VM8       | VM16 | VM24 | VM32        | VM256,
     VM2_n  =       VM2 | VM3 | VM4 | VM5 | VM6 | VM8 | VM9 | VM16 | VM24 | VM32 | VM99 | VM256,
+    VM2_4  =       VM2 | VM3 | VM4,
     VM3_4  =             VM3 | VM4,
     VM3_3n =             VM3 |             VM6       | VM9        | VM24        | VM99 | VM256,
     VM3_n  =             VM3 | VM4 | VM5 | VM6 | VM8 | VM9 | VM16 | VM24 | VM32 | VM99 | VM256,
